### PR TITLE
Adding a support for a handwired split keyboard.

### PR DESCRIPTION
**Pull request by u17194907425**
## Description

Hi. Adding a support for a handwired keyboard that I've designed.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/g_split_board/rules.mk [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+# generated by KBFirmware JSON to QMK Parser
+# https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (fauxpark) [inline keyboards/handwired/g_split_board/rules.mk:25]:
```suggestion
```

File: keyboards/handwired/g_split_board/info.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                {"label":"K42 (D4,E6)", "x":6, "y":4},
+                {"label":"K44 (D4,B5)", "x":8, "y":4},
+                {"label":"K47 (D4,F6)", "x":11, "y":4},
+                {"label":"K49 (D4,B1)", "x":13, "y":4},
+                {"label":"K45 (D4,F4)", "x":9, "y":5},
+                {"label":"K46 (D4,F5)", "x":10, "y":5}
+            ]
+        }
+    }
+    ,"meta": "https://noroadsleft.github.io/kbf_qmk_converter/"
```
Comment (fauxpark) [inline keyboards/handwired/g_split_board/info.json:69]:
```suggestion
```

File: keyboards/handwired/g_split_board/info.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
+{
+    "keyboard_name": "g_split_keyboard",
+    "url": "",
+    "maintainer": "u17194907425",
+    "width": 20,
+    "height": 6.362672937370583,
```
Comment (fauxpark) [inline keyboards/handwired/g_split_board/info.json:6]:
```suggestion
    "height": 6,
```

File: keyboards/handwired/g_split_board/g_split_board.h [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+) { \
+    { K00,   K01,   K02,   K03,   K04,   K05,   K06,   K07,   K08,   K09,   K0A,   K0B   }, \
+    { K10,   K11,   K12,   K13,   K14,   K15,   K16,   K17,   K18,   K19,   K1A,   K1B   }, \
+    { K20,   K21,   K22,   K23,   K24,   K25,   K26,   K27,   K28,   K29,   K2A,   K2B   }, \
+    { K30,   K31,   K32,   K33,   K34,   K35,   K36,   K37,   K38,   K39,   K3A,   K3B   }, \
+    { KC_NO, KC_NO, K42,   K43,   K44,   K45,   K46,   K47,   K48,   K49,   KC_NO, KC_NO }, \
+}
+
+// generated by KBFirmware JSON to QMK Parser
+// https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (fauxpark) [inline keyboards/handwired/g_split_board/g_split_board.h:37]:
```suggestion
```

File: keyboards/handwired/g_split_board/g_split_board.c [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "g_split_board.h"
+
+// generated by KBFirmware JSON to QMK Parser
+// https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (fauxpark) [inline keyboards/handwired/g_split_board/g_split_board.c:20]:
```suggestion
```

File: keyboards/handwired/g_split_board/config.h [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ */
+#define MATRIX_ROW_PINS { D3, D2, D1, D0, D4 }
+#define MATRIX_COL_PINS { C6, D7, E6, B4, B5, F4, F5, F6, F7, B1, B3, B2 }
+
+#define DIODE_DIRECTION COL2ROW
+
+
+// generated by KBFirmware JSON to QMK Parser
+// https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (fauxpark) [inline keyboards/handwired/g_split_board/config.h:49]:
```suggestion
```

File: keyboards/handwired/g_split_board/readme.md [hunk 7]
```diff
@@ -0,0 +1 @@
+A split ergonomic keyboard with enough key's for national layout
```
Comment (fauxpark) [inline keyboards/handwired/g_split_board/readme.md:1]: This file should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md).

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**